doc/source/whatsnew/v2.1.0.rst

```diff
@@ -482,6 +482,7 @@ Styler
 
 Metadata
 ^^^^^^^^
+- Fixed metadata propagation in :meth:`DataFrame.squeeze`, and :meth:`DataFrame.describe` (:issue:`28283`)
```

Comment:
Planning on finishing the rest of the missing methods (only a couple left), so I'll merge all the entries in the future.